docs/concepts/EvictionAndDisruptionBudget/README.md

```diff
@@ -0,0 +1,65 @@
+- The removal of resources from a member cluster by the scheduler due to scheduling policy changes
+- Users manually deleting workload resources running on a member cluster
+- Users manually deleting the `ClusterResourceBinding` object which is an internal resource the represents the placement of resources on a member cluster
+- Workloads failing to run properly on a member cluster due to misconfiguration or cluster related issues.
+
+For all the cases of involuntary disruptions described above, the `Placement Disruption Budget` object does not protect against them.
+
+## ClusterResourcePlacementEviction
+
+An eviction object is used to remove resources from a member cluster once the resources have already been propagated from the hub cluster.
+
+The eviction object is only reconciled once after which it reaches a terminal state. List of terminal states for `ClusterResourcePlacementEviction`,
+- `ClusterResourcePlacementEviction` is valid and it's executed successfully
+- `ClusterResourcePlacementEviction` is invalid
+- `ClusterResourcePlacementEviction` is valid but it's not executed
+
+To successfully evict resources from a cluster, the user needs to specify:
+
+- The name of the `ClusterResourcePlacement` object which propagated resources to the target cluster
+- The name of the target cluster from which we need to evict resources
+
+When specifying the `ClusterResourcePlacement` object in the eviction's spec, the user needs to consider the following cases:
+
+- For `PickFixed` CRP, eviction is not allowed  it is recommended that one directly edit the list of target clusters on the CRP object.
+- For `PickAll` & `PickN` CRPs, eviction is allowed because the users cannot deterministically pick or unpick a cluster based on the placement strategy; it's up to the scheduler.
+
+> **Note:** After an eviction is executed, there is no guarantee that the cluster won't be picked again by the scheduler to propagate resources for a `ClusterResourcePlacement` resource.
+> The user needs to specify a [taint](../../howtos/taint-toleration.md) on the cluster to prevent the scheduler from picking the cluster again. This is especially true for `PickAll ClusterResourcePlacement` because 
+> the scheduler will try to propagate resources to all the clusters in the fleet.
+
+## ClusterResourcePlacementDisruptionBudget
+
+The `ClusterResourcePlacementDisruptionBudget` is used to protect resources propagated by a `ClusterResourcePlacement` to a target cluster from voluntary disruption, i.e., `ClusterResourcePlacementEviction`.
+
+> **Note:** When specifying a `ClusterResourcePlacementDisruptionBudget`, the name should be the same as the `ClusterResourcePlacement` that it's trying to protect.
+
+Users are allowed to specify one of two fields in the `ClusterResourcePlacementDisruptionBudget` spec since they are mutually exclusive:
+
+- MaxUnavailable - specifies the maximum number of clusters in which a placement can be unavailable due to voluntary disruptions.
```

Comment:
It should be `due to any form of disruptions`, right? Currently it reads like `MaxUnavailable` does not count, e.g., failed cluisters, at all. Or more specifically, eviction can only occur if there are less or equal to `MaxUnavailable` # of clusters in the system after the eviction.